pkg/operations/kubernetes162upgrader.go

```diff
@@ -64,14 +66,21 @@ func (ku *Kubernetes162upgrader) RunUpgrade() error {
 	upgradeMasterNode.ResourceGroup = ku.ClusterTopology.ResourceGroup
 	upgradeMasterNode.Client = ku.Client
 
-	// for _, vm := range *ku.ClusterTopology.MasterVMs {
-	// 1.	Shutdown and delete one master VM at a time while preserving the persistent disk backing etcd.
-	upgradeMasterNode.DeleteNode()
-	// 2.	Call CreateVMWithRetries
-	upgradeMasterNode.CreateNode(loopCount)
-	loopCount++
+	// Sort by VM Name (e.g.: k8s-master-22551669-0) offset no. in descending order
+	sort.Sort(sort.Reverse(armhelpers.ByVMNameOffset(*ku.ClusterTopology.MasterVMs)))
```

Comment:
why `sort.Sort(sort.Reverse(...))`?

I could reason about `sort.Reverse(sort.Sort(...))` maybe but not the other way, but I've not used the sort interface before so I'm probably missing it.